services/explorer/src/pages/TransactionData/TransactionData.tsx

```diff
@@ -193,6 +197,32 @@ const TransactionDataContainer = (): JSX.Element => {
                   TransactionType.Contract_Creation) && (
                 <TransactionDataTabs txHash={txHash} />
               )}
+              <Divider />
+              <Typography variant='h6' component='h6' color='primary'>
+                Decoded function data
+              </Typography>
```

Comment:
This will be hidden in the refactored [component](https://github.com/Taraxa-project/taraxa-platform/pull/542/files#diff-4aad2f14d43ab92d4aa7f308f305f9080c581538219ca11e9a28103481c2a8a8R189)